### ISSUE TITLE
3dot popups are still open after navigating further.

### ISSUE THREAD

**Issue by mbacherycz** [Altinn/altinn-components#589]
### Description of the bug

https://github.com/user-attachments/assets/e23b8563-368c-43f2-844c-9c77f76b84ab

### Steps To Reproduce

Should toolbar be closed on loosing focus?
Or maybe an "x" and/or a focus trap should be added?
